Add info on schema integration filters with existing integration object (#5961)
segmentio/segment-docs: src/guides/filtering-data.md

**Integration filters are all-or-nothing for each event.** If you require more detailed control over which events are sent to specific destinations, you can use Destination Filters to inspect the event payload, and conditionally drop the data or forward it to the destination.

**Integration filters will not override an existing value in the integrations object.** If the integration object already has a value for the integration, the per source schema integration filters will not override this. For example, if you are sending events to Appsflyer with the appsflyerId passed into the integration object:

```javascript
integrations: {
  Appsflyer: {
    appsflyerId: 'xxxxxx'
  }
}
```

> And for the same event you have Appsflyer turned off using the per source schema integrations filter, this filter will not override the above object with a false value, and events will still send downstream. In this scenario, [destination filters](#destination-filters) can be used to drop the event before it is sent downstream. 

Reviewer: ```suggestion
For the same event you have Appsflyer turned off using the per source schema integrations filter, this filter won't override the above object with a false value, and events still send downstream. In this scenario, [destination filters](#destination-filters) can be used to drop the event before sends downstream. 
```
